.env

```diff
@@ -27,3 +27,8 @@ REACT_APP_UNSAFE_ERROR_SIMULATION=false
 
 # When true, commenting features will be enabled.
 REACT_APP_ENABLE_COMMENTING=true
+
+# Variables for Google Analytics
+REACT_APP_ANALYTICS_HOST=https://www.google-analytics.com
+REACT_APP_TRACKING_ENABLED=true
+REACT_APP_TRACKING_ID=GA-ID-would-go-here
```

Comment:
Where are the GA ids?

Reply:
We have requested that this site be set up for GA, but that hasn't been completed yet. See https://bugzilla.mozilla.org/show_bug.cgi?id=1613724.

Reply:
We've received the ids now, so I've updated the PR to include them.